services/help-service.ts

```diff
@@ -61,7 +67,34 @@ export class HelpService implements IHelpService {
 	public async generateHtmlPages(): Promise<void> {
 		const mdFiles = this.$fs.enumerateFilesInDirectorySync(this.pathToManPages);
 		const basicHtmlPage = this.$fs.readText(this.pathToBasicPage);
-		await Promise.all(_.map(mdFiles, markdownFile => this.createHtmlPage(basicHtmlPage, markdownFile)));
+		await Promise.all(_.map(mdFiles, markdownFile => this.createHtmlPage(basicHtmlPage, markdownFile, this.pathToManPages, this.pathToHtmlPages)));
+
+		const installedExtensionsData = this.$extensibilityService.getInstalledExtensionsData();
+
+		const basicHtmlPageForExtensions = this.$fs.readText(this.pathToBasicPageForExtensions);
+		for (const extensionData of installedExtensionsData) {
+			const docsDir = extensionData.docs;
+
+			if (docsDir) {
+				this.$logger.trace(`Start generation of html help content for extension ${extensionData.extensionName}`);
+				if (!this.$fs.exists(docsDir)) {
+					this.$logger.warn(`Unable to generate html help pages for extension ${extensionData.extensionName} as the docs directory ${docsDir} does not exist.`);
+					continue;
+				}
+
+				const htmlDirFullPath = path.join(path.dirname(docsDir), "html");
```

Comment:
You can extract `"html"` as a constant either in `constants.ts` or in this file and reuse it here and in `tryOpeningSelectedPage`